internal/goutils/errors.go

```diff
@@ -1,4 +1,4 @@
-package handy
+package goutils
 
 func Must(err error) {
```

Comment:
Can you rename to `MustBeNil`? `Must` is not intuitive.

Reply:
Must is a pattern in golang: https://stackoverflow.com/questions/12434565/built-in-helper-for-must-pattern-in-go
@eberle1080 I think there was some PR a while ago about some other method, and prefixing a method with `Must` that you suggested. What do you think about this case?

Must be nil is a bit confusing to me as it is using negation. Must be empty == without an error == successful. The closest equivalent I can think of is `assert ...` in JAVA.

Ex: Must(loadFile("data.json"))

Reply:
Maybe we could take inspiration from this stackoverflow or blog: https://www.willem.dev/code-snippets/generic-must-function/
To imrpove the `Must` method.

Reply:
See https://github.com/amp-labs/connectors/pull/1114#discussion_r1830437721